reana_client/cli/files.py

```diff
@@ -207,10 +210,20 @@ def download_files(
 
     Examples: \n
     \t $ reana-client download # download all output files \n
-    \t $ reana-client download mydata.tmp outputs/myplot.png
+    \t $ reana-client download mydata.tmp outputs/myplot.png \n
+    \t $ reana-client download -o - data.txt # write data.txt to stdout
     """
     from reana_client.api.client import download_file, get_workflow_specification
 
+    def display_files_content(binary_file, multiple_files_zipped):
```

Comment:
Could you please add a docstring here?

**Note.** Since we've dropped Python 2 support some months ago, we're starting to gradually include [type hints](https://docs.python.org/3/library/typing.html) to our functions.

In this case, you could use something like this:

```python
def display_files_content(binary_file: bytes, multiple_files_zipped: boolean) -> None:
```